ddtrace/internal/rand.py

```diff
@@ -0,0 +1,46 @@
+import threading
+
+from ddtrace import compat
+
+if compat.PY3:
+
+    def randbits_gen():
+        while True:
+            yield compat.getrandbits(64)
+
+    gen = randbits_gen()
+
+    def get_rand64_gen():
+        return gen
+
+
+else:
+
+    try:
+        from ._rand import xorshift64s, get_rand64_gen
```

Comment:
since this is a required extension, do we actually ever expect this to not be available?

Falling back to a pure-Python version I know is still even better than the built-in Python code, but it almost seems like it'll be a bug if someone hits this.